### PR TITLE
Removed blue header

### PR DESCRIPTION
**Pull request by nparsons08**
Removed blue beta banner from the application.

### CODE REVIEW

**Comment by kenhoff** [comment]
might also need to adjust the `.app` padding in `_app.scss`, lines 12-14

**Comment by nparsons08** [comment]
@kenhoff Thanks, didn't catch that.

File: app/src/styles/framework/_app.scss [hunk 1]
```diff
@@ -8,10 +8,9 @@ body {
 }
 
 .app {
-	// this is just for a little while, while we're in prerelease mode.
 	-webkit-app-region: drag;
 	padding-top: 12em;
```
Comment (kenhoff) [inline app/src/styles/framework/_app.scss]: probably only need one `padding-top` declaration here

**Comment by kenhoff** [comment]
\o/

**Pull request merged**